### DISCUSSION TITLE
Avoiding Overlap of polygons

### DISCUSSION THREAD

**Discussion by igorleite91** [tmizu23/BezierEditing#12] [Q&A]
Is there any way that when I draw one polygon on top of the other it doesn't overlap? 

![over](https://user-images.githubusercontent.com/49123452/116802079-529aaa80-aae6-11eb-9292-50bc32b74610.PNG)

Because the option to add qgis native polygon allows there to be no over positioning by checking this option

![over2](https://user-images.githubusercontent.com/49123452/116802100-8fff3800-aae6-11eb-9b01-696a9bb62523.PNG)

Please help me to solve this problem. I have a giant mapping and I need some tips. Your tool is wonderful, but I was wondering how to fix this from the overlay.

**Comment by tmizu23** [comment]
Thanks for the question.

I think it would be useful to be able to do that, but implementing topology-aware Bezier Editing is difficult, so I can't do it right away.

If I want to create a topology-aware polygon, I first create one polygon for the entire range, and then divide it by lines written in Bezier Eding.

**Reply by igorleite91** [reply]
Thank you very much for the agility in responding and for the tip regarding the creation of the polygon and clipping with bezier lines. I understand how difficult it really must be to question the topology of polygons.

Thank you very much!
